environments/.stackhpc/SMS.pkrvars.hcl

```diff
@@ -0,0 +1,10 @@
+flavor = "general.v1.small"
+use_blockstorage_volume = true
+volume_size = 15 # GB
+image_disk_format = "qcow2"
```

Comment:
I think we should move these three to Packer default values since their set to the same value for all three clouds.